sdk/network/Azure.ResourceManager.Network/src/autorest.md

```diff
@@ -7,8 +7,8 @@ Run `dotnet build /t:GenerateCode` to generate code.
 azure-arm: true
 library-name: Network
 namespace: Azure.ResourceManager.Network
-require: https://github.com/Azure/azure-rest-api-specs/blob/738879cc6e1c5569b01130fd69a2587388fc34b3/specification/network/resource-manager/readme.md
-# tag: package-2024-03
+require: https://github.com/Azure/azure-rest-api-specs/blob/04c21fe3f2870bc33a397787dc377495f33b3d82/specification/network/resource-manager/readme.md
```

Comment:
The spec hasn't been merged into main

Reply:
we are still waiting for NRP release. will draft this PR until then

Reply:
https://github.com/Azure/azure-rest-api-specs/blob/5c9d5f957d76d9fea9c513f494660c6c5d3e809a/specification/network/resource-manager/readme.md

Is this ready?